packages/app-project/src/shared/components/ProjectStatistics/ProjectStatisticsContainer.spec.js

```diff
@@ -1,62 +1,58 @@
-import { shallow } from 'enzyme'
-
-import ProjectStatistics from './ProjectStatistics'
-import { ProjectStatisticsContainer } from './ProjectStatisticsContainer'
-
-let wrapper
-let projectStatisticsWrapper
-
-const CLASSIFICATIONS = 1
-const COMPLETED_SUBJECTS = 2
-const ROUTER = {
-  asPath: '/foo/bar',
-  query: {
-    owner: 'foo',
-    project: 'bar'
-  }
+import { render, screen, waitFor } from '@testing-library/react'
+import { within } from '@testing-library/dom'
+import { composeStory } from '@storybook/react'
+import Meta, { ProjectStatistics } from './ProjectStatisticsContainer.stories.js'
+import { ProjectStatisticsContainerMock, ProjectStatisticsContainerRouterMock } from './ProjectStatisticsContainer.mock'
+import Router from 'next/router'
+import { RouterContext } from 'next/dist/shared/lib/router-context.shared-runtime'
+import { format } from 'd3'
+
+function RouterMock({ children }) {
+  Router.router = ProjectStatisticsContainerRouterMock
+
+  return (
+    <RouterContext.Provider value={ProjectStatisticsContainerRouterMock}>
+      {children}
+    </RouterContext.Provider>
+  )
 }
-const SUBJECTS = 3
-const VOLUNTEERS = 4
-
-describe('Component > ProjectStatisticsContainer', function () {
-  before(function () {
-    wrapper = shallow(<ProjectStatisticsContainer
-      classifications={CLASSIFICATIONS}
-      completedSubjects={COMPLETED_SUBJECTS}
-      router={ROUTER}
-      subjects={SUBJECTS}
-      volunteers={VOLUNTEERS}
-    />)
-    projectStatisticsWrapper = wrapper.find(ProjectStatistics)
-  })
 
-  it('should render without crashing', function () {
-    expect(wrapper).to.be.ok()
+describe.only('Component > ProjectStatisticsContainer', function () {
```

Comment:
Stray `.only()` here.
```suggestion
describe('Component > ProjectStatisticsContainer', function () {
```